nautobot/extras/api/serializers.py

```diff
@@ -892,7 +892,7 @@ class Meta:
 class StatusModelSerializerMixin(serializers.Serializer):
     """Mixin to add non-required `status` choice field to model serializers."""
 
-    status = StatusSerializerField(required=False, queryset=Status.objects.all())
+    status = StatusSerializerField(queryset=Status.objects.all(), required=True)
```

Comment:
`required=True` is the default for a serializer field, so it can be omitted, and I think doing so would be more consistent style-wise as I don't see a single serializer field in our code base explicitly declaring `required=True` otherwise.

```suggestion
    status = StatusSerializerField(queryset=Status.objects.all())
```